cpp/test/neighbors/ann_cagra_c.cu

```diff
@@ -66,6 +80,11 @@ TEST(CagraC, BuildSearch)
   cuvsCagraIndexParamsCreate(&build_params);
   cuvsCagraBuild(res, build_params, &dataset_tensor, index);
 
+  cuvsCagraExtendParams_t extend_params;
+  cuvsCagraExtendParamsCreate(&extend_params);
```

Comment:
We should probably cleanup the memory here w/ `cuvsCagraExtendParamsDestroy` at the end